core/domain/feedback_domain.py

```diff
@@ -19,11 +19,31 @@
 
 
 class FeedbackThread(object):
-    """Domain object for a feedback thread."""
+
+    """Domain object for a feedback thread.
+
+    Attributes:
+        full_thread_id: str. The feedback thread ID.
+        exploration_id: str. The associated exploration's ID.
+        state_name: str. The name of the state associated with
+            the feedback thread.
+        original_author_id: str. The ID of the original author.
+        status: str. The current status of the thread. Status should
+            be one of core.storage.feedback.gae_models.STATUS_CHOICES.
+        subject: str. The subject of the feedback thread.
+        summary: str. A summary of the feedback thread.
+        has_suggestion: bool. Whether the feedback thread includes a
+            suggestion.
+        created_on: str. The date on which the feedback thread was created.
```

Comment:
Thank you for showing me how to use logging to figure the type out, I didn't know I could do that! I have fixed this, correct type is datetime.datetime.